src/structures/Base.ts

```diff
@@ -27,7 +28,7 @@ export default class Base {
    * @return {*}  {Promise<any>}
    * @memberof Base
    */
-  async request (method: Method, uri: string, data?: object): Promise<any> {
+  async request(method: Method, uri: string, data?: object): Promise<any> {
```

Comment:
I should remove the 'any' 🤔

Reply:
It could have different properties depending of the API response, so I don't know typing it correctly, maybe "unknown" could be a solution.
All the other suggestions have been commited.